resources/asciidoctor/lib/chunker/breadcrumbs.rb

```diff
@@ -11,21 +11,31 @@ module Breadcrumbs
     def generate_breadcrumbs(doc, section)
       result = ['<div class="breadcrumbs">']
       result += generate_breadcrumb_links(section).reverse
-      result << %(<span class="breadcrumb-node">#{section.title}</span>)
       result << '</div>'
       Asciidoctor::Block.new doc, :pass, source: result.join("\n")
     end
 
     def generate_breadcrumb_links(section)
       result = []
       parent = section
+      first = true
       while (parent = parent.parent)
         extra = parent.context == :document ? parent.attr('title-extra') : ''
-        result << <<~HTML.strip
+        first_link = <<~HTML.strip
           <span class="breadcrumb-link"><a #{link_href parent}>#{parent.title}#{extra}</a></span>
-          »
         HTML
+        next_links = first_link + <<~HTML.strip
+          <span class="material-symbols-outlined">chevron_right</span>
+        HTML
+        # This prevents a chevron from being placed after the last breadcrumb
+        links = first == true ? first_link : next_links
+        result << links
+        first = false
       end
+      result << <<~HTML.strip
+        <span class="breadcrumb-link"><a href="/guide/">Docs home</a></span>
```

Comment:
@gchaps / @kellyemurphy Do you have any thoughts on what words we should use to describe the link to elastic.co/guide?

Reply:
I vote for "Elastic Docs" because it matches the title of the  page. "Elastic Docs Home" would also be good because it's more descriptive, but it makes the text longer.

"Home" on its own could mean the home page for the whole Elastic site, so I would avoid that.

Reply:
The new, less-cluttered look to the page is so much easier to read!

+1 for Elastic Docs but shouldn't it be sentence case:  Elastic docs.  Which means we should also change the docs landing page title to "Welcome to Elastic docs".  

Also "Observability guide"

Reply:
"Observability Guide" comes from [the docs themselves](https://github.com/elastic/observability-docs/blob/main/docs/en/observability/index.asciidoc?plain=1#L3), unless we add some code that automatically sentence-cases everything in the breadcrumbs (which we'd likely need to maintain a list of exceptions for).

Reply:
Adding to Greg's comment. I went with title case for Observability docs because all of our book titles use title case:

<img width="748" alt="Screen Shot 2022-07-12 at 8 03 40 AM" src="https://user-images.githubusercontent.com/5618806/178522235-ffb29627-e4d6-4e30-a28b-b9904a9ebbab.png">